app/models/invoice.rb

```diff
@@ -11,6 +11,10 @@ class Invoice < ApplicationRecord
 
   before_save :ensure_organization_sequential_id, if: -> { organization.per_organization? }
```

Comment:
Can we please move the code in the invoice model (everything except has_many definition) into separate PR that would be in the draft status until we ship and QA the feature? I would like to avoid calling any new code before the real QA of the feature. Probably we can continue using workflow as before where we would merge only dead code which is not called in any way :)

Reply:
I created a separate [PR](https://github.com/getlago/lago-api/pull/2012)